proto/dymensionxyz/dymension/sequencer/tx.proto

```diff
@@ -53,4 +56,17 @@ message MsgUnbond {
 // MsgUnbondResponse defines the Msg/Unbond response type.
 message MsgUnbondResponse {
   google.protobuf.Timestamp completion_time = 1 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
-}
\ No newline at end of file
+}
+
+// MsgIncreaseBond defines a SDK message for increasing the bond amount of a sequencer.
+message MsgIncreaseBond {
+  option (cosmos.msg.v1.signer) = "creator";
+  // creator is the bech32-encoded address of the sequencer account which is the account that the message was sent from.
+  string creator = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  
+  // amount is the amount of coins to be added to the sequencer's bond.
+  cosmos.base.v1beta1.Coin amount = 2 [(gogoproto.nullable) = false];
```

Comment:
Like I commented in the ADR, perhaps name this `add_amount` or something like that, to avoid ambiguity?